aws-lambda/jaeger-ruby/prime_sum.rb

```diff
@@ -0,0 +1,78 @@
+# This simple Lambda example using a Ruby handler takes a request body with
+# {
+#   body: "Some message",
+#   queryStringParameters: {
+#     limit: 100
+#   }
+# }
+# and returns the sum of all prime numbers below 'limit'.
+
+require 'opentracing'
+require 'signalfx/lambda'
+
+# this is the original lambda handler
+def handle_request(event:, context:)
+  sum = 0
+
+  # when using the wrapper, the tracer is available through OpenTracing.global_tracer
+  # to do more granular instrumentation
+  OpenTracing.global_tracer.start_active_span("handle_request") do |scope|
```

Comment:
This span seems superfluous since the wrapper will effectively have a similar span.